src/mscorlib/shared/System/IO/Path.Unix.cs

```diff
@@ -193,10 +193,15 @@ public static bool IsPathRooted(string path)
             return path.Length > 0 && path[0] == PathInternal.DirectorySeparatorChar;
         }
 
+        // The resulting string is null if path is null. If the path is empty or
+        // only contains whitespace characters an ArgumentException gets thrown.
         public static string GetPathRoot(string path)
         {
             if (path == null) return null;
-            return IsPathRooted(path) ? PathInternal.DirectorySeparatorCharAsString : String.Empty;
+			if (string.IsNullOrWhiteSpace(path))
+                throw new ArgumentException(SR.Arg_PathIllegal);
```

Comment:
On ReferenceSources the param isn't used: https://referencesource.microsoft.com/#mscorlib/system/io/path.cs,431

But adding it shouldn't break anything, right?